tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -302,3 +331,54 @@ def test_plot_pareto_front_invalid_axis_order(
             include_dominated_trials=include_dominated_trials,
             axis_order=invalid_axis_order,
         )
+
+
+def test_plot_pareto_front_targets_without_target_names() -> None:
+    study = optuna.create_study(directions=["minimize", "minimize", "minimize"])
+    with pytest.raises(ValueError):
```

Comment:
How about specifying the expected error message with `match` argument because this method has several conditions to raise `ValueError`? This might be helpful for other contributors.
```suggestion
    with pytest.raises(
        ValueError,
        match="If `targets` is specified for empty studies, `target_names` must be specified.",
    ):
```